p2p/src/net/default_backend/tests.rs

```diff
@@ -66,7 +66,7 @@ where
     {
         assert_eq!(address, conn2.local_addresses()[0]);
         assert_eq!(peer_info.network, *config.magic_bytes());
-        assert_eq!(peer_info.version, SemVer::new(0, 1, 0));
+        assert_eq!(peer_info.version, *config.version());
```

Comment:
The version should be coming from Cargo... not from config. But maybe this is something for another day.

You can write a TODO in ChainConfig at the version variable stating this.

Reply:
I updated `Builder` to use `CARGO_PKG_VERSION`. Is this OK?
```
...
            version: SemVer::try_from(env!("CARGO_PKG_VERSION"))
                .expect("invalid CARGO_PKG_VERSION value"),
```